packages/badge/src/Badge.tsx

```diff
@@ -16,54 +16,56 @@ export const Variant = {
 export type Variant = typeof Variant[keyof typeof Variant];
 
 export const baseStyle = css`
+  font-family: Euclid Circular A, ‘Helvetica Neue’, Helvetica, Arial, sans-serif; // TODO: Refresh – remove when fonts are updated
   display: inline-flex;
   align-items: center;
-  font-weight: bold;
-  font-size: 11px;
-  line-height: 20px;
-  border-radius: 50px;
-  height: 20px;
-  padding-left: 9px;
-  padding-right: 9px;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  border-radius: 5px;
+  height: 18px;
+  padding-left: 6px;
+  padding-right: 6px;
   text-transform: uppercase;
   border: 1px solid;
+  letter-spacing: 1px;
 `;
 
 export const badgeVariants: { [K in Variant]: string } = {
   [Variant.LightGray]: css`
-    background-color: ${uiColors.gray.light3};
-    border-color: ${uiColors.gray.light2};
-    color: ${uiColors.gray.dark1};
+    background-color: ${palette.gray.light3};
+    border-color: ${palette.gray.light2};
+    color: ${palette.gray.dark1};
   `,
 
   [Variant.DarkGray]: css`
-    background-color: ${uiColors.gray.dark2};
-    border-color: ${uiColors.gray.dark3};
-    color: ${uiColors.white};
+    background-color: ${palette.gray.dark2};
+    border-color: ${palette.gray.dark3};
+    color: ${palette.white};
```

Comment:
Maybe let the design team know this will still be called `DarkGray` and not `White`

Reply:
Also, not sure why, but this is how it appears when I first open Storybook. If I go to a different variant and come back to DarkGray it shows correctly 
<img width="89" alt="Screen Shot 2022-01-24 at 1 32 54 PM" src="https://user-images.githubusercontent.com/2414030/150843494-ce526138-8766-431a-90d9-6e2f503db631.png">

Reply:
Seems like it was just a cache issue. Resolved now